package.json

```diff
@@ -38,11 +35,10 @@
     "test-changed": "lerna run test --stream --concurrency 1 --since origin/HEAD",
     "test-ci": "lerna run test-ci --concurrency 1",
     "test-ci-electron": "lerna run test-ci-electron --concurrency 1",
-    "pretest-packages-publish": "./scripts/monorepo/start-proxy-registry.sh && wait-on -t 3000 http://localhost:4873 && npm-cli-adduser -u test -p test -e test@test.test --registry http://localhost:4873 && echo $'\\n\\n!!! LOCAL REGISTRY DOCKER IMAGE IS RUNNING, DO NOT FORGET TO CLEAN IN UP WHEN YOU ARE DONE\\n\\n'",
-    "test-packages-publish": "lerna publish from-package --registry http://localhost:4873",
-    "test-packages-version": "lerna version --no-push --no-private -m \"tmp(release): Test release\"",
     "update-authors": "node scripts/generate-authors.js",
     "update-third-party-notices": "node scripts/gather-licenses.js packages/compass/ > THIRD-PARTY-NOTICES.md",
+    "bump-packages": "bump-monorepo-packages",
+    "publish-packages": "lerna publish from-package --no-verify-access --no-changelog --no-push --exact --no-git-tag-version --force-publish --yes",
```

Comment:
I might be missing something, but there are three flags here that we probably should remove to avoid confusion:

`--force-publish` is used by version command to force bump every package disregarding if it was changed or not
`--exact` is used by version command to update dependents to the exact version of the bumped package

as we are doing `publish from-package` these two are not applicable anyway, but are confusing because their purpose is the reverse of how we are bumping the versions and detecting changes

`--no-changelog` is a version command flag that only takes effect if conventional commit functionality of lerna is used, and we are not using it

Reply:
I'm not sure why those are there, I copied all of this from `mongosh`, and to my understanding when `from-package` is used all of the `lerna version` options should not take effect, but I'm a bit afraid that some of them are there cause they were necessary. I guess I can try to remove them.. worse case we will also be able to test the manual run  :)

Reply:
Did the same for the other repo: https://github.com/mongodb-js/devtools-shared/pull/29

Reply:
Yeah, I mean if they are there for a reason, let's totally keep them, but if we are not sure, I would prefer to try out without them first / figure out if they are needed or not

Reply:
I think we should be able to test it on the other repo, just to be safe